sdks/python/apache_beam/runners/dataflow/ptransform_overrides.py

```diff
@@ -236,7 +236,10 @@ def enter_composite_transform(self, transform_node):
         self.visit_transform(transform_node)
 
       def visit_transform(self, transform_node):
-        if [o for o in self.outputs if o in transform_node.inputs]:
+        # Internal consumers of the outputs we're overriding are expected.
+        # We only error out on non-internal consumers.
+        if ('BigQueryBatchFileLoads' not in transform_node.full_label and
```

Comment:
Can we use the experiment here ? You can get hold of the PipelineOptions through input.

Reply:
What would we use the experiment for? In this case, the experiment is absent, so we would not need it.